### ISSUE TITLE
Can no longer run DinD - Error starting daemon: error initializing graphdriver

### ISSUE THREAD

**Issue by tomdee** [docker/docker#14543]
I see the following error trying to start docker inside docker
FATA[0000] Error starting daemon: error initializing graphdriver: Error found less than 3 fields post '-' in "193 192 0:54 / /sys/fs/cgroup rw,nosuid,nodev,noexec,relatime - tmpfs  rw"

$ docker version
```
Client:
 Version:      1.8.0-dev
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   b7e8169
 Built:        Fri Jul 10 21:20:22 UTC 2015
 OS/Arch:      linux/amd64
 Experimental: true

Server:
 Version:      1.8.0-dev
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   b7e8169
 Built:        Fri Jul 10 21:20:22 UTC 2015
 OS/Arch:      linux/amd64
 Experimental: true
```

cgroups outside:
```
cgroup on /sys/fs/cgroup/blkio type cgroup (rw,nosuid,nodev,noexec,relatime,blkio)
cgroup on /sys/fs/cgroup/cpu,cpuacct type cgroup (rw,nosuid,nodev,noexec,relatime,cpu,cpuacct)
cgroup on /sys/fs/cgroup/cpuset type cgroup (rw,nosuid,nodev,noexec,relatime,cpuset,clone_children)
cgroup on /sys/fs/cgroup/devices type cgroup (rw,nosuid,nodev,noexec,relatime,devices)
cgroup on /sys/fs/cgroup/freezer type cgroup (rw,nosuid,nodev,noexec,relatime,freezer)
cgroup on /sys/fs/cgroup/hugetlb type cgroup (rw,nosuid,nodev,noexec,relatime,hugetlb)
cgroup on /sys/fs/cgroup/memory type cgroup (rw,nosuid,nodev,noexec,relatime,memory)
cgroup on /sys/fs/cgroup/net_cls,net_prio type cgroup (rw,nosuid,nodev,noexec,relatime,net_cls,net_prio)
cgroup on /sys/fs/cgroup/perf_event type cgroup (rw,nosuid,nodev,noexec,relatime,perf_event)
cgroup on /sys/fs/cgroup/systemd type cgroup (rw,nosuid,nodev,noexec,relatime,xattr,release_agent=/lib/systemd/systemd-cgroups-agent,name=systemd)
tmpfs on /sys/fs/cgroup type tmpfs (ro,nosuid,nodev,noexec,mode=755)
```

cgroups inside:
```
cgroup on /sys/fs/cgroup/blkio type cgroup (ro,relatime,blkio)
cgroup on /sys/fs/cgroup/cpu,cpuacct type cgroup (ro,relatime,cpu,cpuacct)
cgroup on /sys/fs/cgroup/cpuset type cgroup (ro,relatime,cpuset,clone_children)
cgroup on /sys/fs/cgroup/devices type cgroup (ro,relatime,devices)
cgroup on /sys/fs/cgroup/freezer type cgroup (ro,relatime,freezer)
cgroup on /sys/fs/cgroup/hugetlb type cgroup (ro,relatime,hugetlb)
cgroup on /sys/fs/cgroup/memory type cgroup (ro,relatime,memory)
cgroup on /sys/fs/cgroup/name=systemd type cgroup (ro,relatime,xattr,release_agent=/lib/systemd/systemd-cgroups-agent,name=systemd)
cgroup on /sys/fs/cgroup/net_cls,net_prio type cgroup (ro,relatime,net_cls,net_prio)
cgroup on /sys/fs/cgroup/perf_event type cgroup (ro,relatime,perf_event)
```

**Comment by jhowardmsft** [comment]
@tomdee - Could #13312 explain it? As I mentioned on IRC, I'm somewhat doubtful it's the PR's I submitted recently causing this, as Jenkins runs dind, and the Windows code path doesn't touch anything on the cgroup path.

**Comment by tomdee** [comment]
Ah yes, that looks like a more likely candidate. I won't be able to test (without) it until Monday though.

**Comment by jhowardmsft** [comment]
#microsoft-is-not-evil-afterall ;)  OK, been a long week!

**Comment by hqhq** [comment]
Looks likely my PR causes this. Maybe dind want to write something to /sys/fs/cgroup/ in container, I'm afraid I won't be able to fix it until Monday.

**Comment by thaJeztah** [comment]
> #microsoft-is-not-evil-afterall

LOL, @jhowardmsft 

![image](https://cloud.githubusercontent.com/assets/1804568/8633016/e259f21c-27b5-11e5-9879-e705a5f74e5a.jpg)

**Comment by hqhq** [comment]
Sorry about the regression, I don't think there is a quick fix or work around, so I sent a PR to revert this, https://github.com/docker/docker/pull/14575

**Comment by hqhq** [comment]
I confirmed with fix from https://github.com/opencontainers/runc/pull/127, dind is working again, I read `wrapdocker` in dind image, it won't write anything to /sys/fs/cgroup, I'm not sure if anywhere else need to write to this dir, but this problem is fixed.

**Comment by hqhq** [comment]
This issue can be closed after #14693 is merged.
